Documentation/Signoffs/Sign.md

```diff
@@ -0,0 +1,96 @@
+# Sign System Detail Design
+
+
+
+## Function of the Subsystem
+
+The sign that will be implemented into this project will show the current number of available parking spots for the monitored lot.This will be updated roughly as often as the mobile application. The data being sent to the display will be from the primary and secondary data acquisition systems. That data will then be sent to the server and programmed into readable data to the sign. The sign will then consist of a text message stating “Open Spots Available”.Then, there will be two large weatherproof seven segment displays stating how many available spots there are. This will then be controlled by an Arduino that is wirelessly receiving data from the server.This whole system will then need to be waterproofed and mounted to a standing sign.
+
+## Constraints
+
+1. Seven-Segment Display & Controls
+  -At least 1 foot in height and 6 inches in width (LED display)
+  -Microprocessor to control each LED and receive data
+
+2. Wireless Data Transfer Module to receive information
+
+3. Outdoor Physical Sign/Stand
+  -Text above Seven-Segment Display Stating “Open Spots Available”
+
+4. 12 V DC Power for LEDs and Arduino
+
+
+## Buildable Schematic
+
+![Figure 2. Physical Sign Schematic with LED Strip Formation](../Images/Sign_Buildable_Schematic.PNG)
```

Comment:
No 3d schematic. The electrical schematic software needs to match across all designs. Discuss as a team which to use. This style is hard to use and does not scale well. 

Most importantly, this would not work (seems unrealistic to think a low power MCU will provide current to LEDs that are visible at a distance). Nor does it deal with any of the powering and communication distance.